### ISSUE TITLE
Support for Indian Regional languages like Hindi, Kannada, Telugu, Bengali, etc in terra-aggregate-translations.

### ISSUE THREAD

**Issue by shirshandu** [cerner/terra-aggregate-translations#28]
# Feature Request

## Description

I wanted to check for the support for Indian Regional languages like Hindi, Kannada, Telugu, Bengali, etc in terra-aggregate-translations.

All the above languages are part of react-intl when I check its locale-data exists in the library 

When I pass hi/kn(Hindi/Kannada) to the array in i18nSupportedLocales.js it does renders the languague.

## @ Mentions

@ShettyAkarsh @lokesh-0813

**Comment by ryanthemanuel** [comment]
@shirshandu our current supported language/locale list is driven by the most commonly used languages/locales that Cerner supports in its clinical applications. The translations team does not have the capacity or funding to do broad translations (for something like all of Terra) beyond that. If you have a need that's not in our default list, we do support teams using a custom language. You would just need to specify all of the locales you require in your i18n config file and then provide translations for everything in the translations directory.

**Comment by shirshandu** [comment]
Thanks, @ryanthemanuel, can I get any documentation that helps me to understand how to implement how to add languages which are not in the default list of locale.

I am using my own webpack config so probably I need to configure there itself.

**Comment by ryanthemanuel** [comment]
It should be fairly simple, you will just need to list your additional languages in the i18n config file: https://github.com/cerner/terra-aggregate-translations#terrai18nconfig-example  You will then be able to place any translations you need in a top level translations directory (similar to https://github.com/cerner/terra-dev-site/tree/main/translations).  Aggregate translations will automatically warn about strings that are missing so you should be able to figure out what you need to translate from those warnings.

**Comment by ryanthemanuel** [comment]
Closing this for now.  @shirshandu reach out if you have any trouble getting this to work.
